tests/test_metrics_per_city.py

```diff
@@ -0,0 +1,32 @@
+from city_metrix.metrics import *
+from tests.resources.bbox_constants import GEOZONE_TERESINA_WGS84
+
+# TODO - Consider adding other metrics
+
+def test_write_BuiltLandWithHighLST():
+    metric_obj = BuiltLandWithHighLST()
+    metric_values = metric_obj.get_metric(geo_zone=GEOZONE_TERESINA_WGS84)
+    _evaluate_metric_values(metric_values, 2, 0, 0.11, 0.04, 138)
+
+
+def test_write_NaturalAreasPercent():
+    metric_obj = NaturalAreasPercent()
+    metric_values = metric_obj.get_metric(geo_zone=GEOZONE_TERESINA_WGS84)
+    _evaluate_metric_values(metric_values, 2, 1.96, 97.16, 36.91, 138)
+
+
+def _evaluate_metric_values(metric_values, digits, expected_min, expected_max, expected_mean, expected_count):
```

Comment:
Could add the catch and count of NA values? Small sub-regional areas may also have NAs for some indicators.